Compiler/src/ssair/inlining.jl

```diff
@@ -1625,14 +1623,14 @@ function assemble_inline_todo!(ir::IRCode, state::InliningState)
     todo = Pair{Int, Any}[]
 
     for idx in 1:length(ir.stmts)
-        flag = ir.stmts[idx][:flag]
+        inst = ir.stmts[idx]
+        flag = inst[:flag]
+        info = inst[:info]
```

Comment:
looks like `process_simple!` can call `inline_apply!` which contains `ir.stmts[idx][:info] = new_info`, so maybe the variable assignment `info = inst[:info]` needs to still happen on L1634 ?

Reply:
That was it, thanks a lot for the suggestion!